hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/snapshot/SnapshotDiffManager.java

```diff
@@ -217,37 +270,91 @@ public SnapshotDiffReport getSnapshotDiffReport(final String volume,
 
       Map<String, String> tablePrefixes =
           getTablePrefixes(toSnapshot.getMetadataManager(), volume, bucket);
-
-      final Set<String> deltaFilesForKeyOrFileTable =
-          getDeltaFiles(fromSnapshot, toSnapshot,
-              Collections.singletonList(fsKeyTable.getName()), fsInfo, tsInfo,
+      List<String> tablesToLookUp =
+              Collections.singletonList(fsKeyTable.getName());
```

Comment:
Looks like we will never need to explicitly add `fileTable` here since `getKeyTable()` auto picks the right one. In this case, can we just use `String` rather than a `List`?

Write more code comments inline.

Reply:
Will do this as part of the refactoring PR.